manager-service/src/main/java/cz/fi/muni/pa165/hockeymanager/service/HockeyPlayerService.java

```diff
@@ -1,4 +1,55 @@
 package cz.fi.muni.pa165.hockeymanager.service;
 
+import cz.fi.muni.pa165.hockeymanager.entity.HockeyPlayer;
+import cz.fi.muni.pa165.hockeymanager.entity.Match;
+import cz.fi.muni.pa165.hockeymanager.entity.Team;
+
+import java.util.List;
+
+/**
+ * @author Petr Šopf (506511)
+ */
 public interface HockeyPlayerService {
```

Comment:
should be annotated with @Service .

Reply:
Are you sure? Even the interface, not just the implementation?